components/automate-chef-io/content/docs/iam-v2-guide.md

```diff
@@ -221,4 +258,119 @@ Should you have a need to revert back to IAM v1, it is quick and easy to do:
+
+To tag any ingested resource with a project, that project must have at least one rule. A project rule contains conditions that
+determine if an ingested resource should be tagged with the rule's associated project. Each condition contains an attribute, operator, and value. See the [API reference]({{< relref "iam-v2-api-reference.md#project-rules" >}}) for details on how to manage project rules.
+Take this rule for the project `project-devops` as an example:
+
+```json
+{
+  "id": "devops-rule",
+  "name": "devops rule",
+  "type": "NODE",
+  "project_id": "project-devops",
+  "conditions": [
+    {
+      "operator": "EQUALS",
+      "attribute": "CHEF_ORGANIZATION",
+      "values": [
+        "devops"
+      ]
+    }
+  ]
+}
+```
+
+Once this project rule has been created, it must be applied in order to take effect. Once the project rule is in effect, all ingested resources with a `CHEF_ORGANIZATION` field that has the value `devops` will be considered a part of the `project-devops` project.
+Only these resources will appear in Automate when the `project-devops` project has been selected in the global project filter.
+
+{{% warning %}}
+Compliance reports must be using audit cookbook 7.5+ in order to make use of all of the available project rule attributes. Older reports will only have environment and role available as attributes.
+{{% /warning %}}
+
+In this example you will add a project rule to a project you just created. You will apply this new project rule so that all matching ingested resources get tagged with the appropriate project. You will then grant a user access to only that project
+and log in to that user's account to see their filtered view.
+
+First, determine which ingested resources should belong to the project. In this example, we want to tag Compliance reports
+with the Chef Organization `devops`, Infrastructure nodes with the environment `dev`, and actions on Chef Server `devops.pizza` with the project `project-devops`.
+You may want to verify that those filters work as expected using the search filter bars on the Event Feed, Client Runs, and
+Reports pages.
+
+{{% info %}}
+It is important to note that Compliance and Infrastructure resources will not necessarily match under the same conditions.
+Separate conditions governing Compliance and Infrastructure resources should be used. Similarly, events require conditions of `EVENT` type to be tagged correctly. A condition of type `NODE` will not match an event, even if the condition's
+operator, attribute, and value all match (and vice versa with `EVENT` project rules and node).
+{{% /info %}}
+
+Navigate to the project details page of `project-devops`, created in the previous section, by heading to `https://{{< example_fqdn "automate" >}}/settings/projects/project-devops` or from the project list page.
```

Comment:
can we anchor link to the mention of _previous section_ instead of saying _previous section_?